nengo_loihi/splitter.py

```diff
@@ -180,13 +180,7 @@ def split(model, inter_rate, inter_n):  # noqa: C901
                 with host:
                     max_rate = inter_rate * inter_n
                     assert max_rate <= 1000
-
-                    logger.debug("Creating NIF ensemble for %s", c)
-                    ens = nengo.Ensemble(
-                        2 * dim, dim, neuron_type=NIF(tau_ref=0.0),
-                        encoders=np.vstack([np.eye(dim), -np.eye(dim)]),
-                        max_rates=[max_rate] * dim + [max_rate] * dim,
-                        intercepts=[-1] * dim + [-1] * dim)
+                    # TODO: move the above check elsewhere
```

Comment:
Ooh, this looks weird now since you can't actually see the check (unless you go commit-by-commit). The check was that `max_rate <= 1000`, where `max_rate = inter_rate * inter_n`.

Reply:
Yeah, I just kept that check from before, and it doesn't seem right to me.  It also seems like something that should be looked into in #55 .  I've added a TODO here and made a note about it in that PR